src/spectrecoff-cli/commands/Grid.fs

```diff
@@ -0,0 +1,78 @@
+﻿namespace SpectreCoff.Cli.Commands
+
+open Spectre.Console
+open Spectre.Console.Cli
+open SpectreCoff
+
+type GridSettings()  =
+    inherit CommandSettings()
+
+type GridExample() =
+    inherit Command<GridSettings>()
+    interface ICommandLimiter<GridSettings>
+
+    override _.Execute(_context, _settings) =
+        let numbersGrid = grid [
+            Numbers [1; 2]
+            Strings ["One"; "Two"; "Three"]
+        ]
+        Many [
+            C "The grid will have as many columns as are needed to accomodate the longest row:"
+            numbersGrid.toOutputPayload
+        ]
+        |> toConsole
```

Comment:
In all other places, we have pulled this call up to the prev line. (`] |> toConsole`). I find that a bit more compact, as I think the poor line with only the closing `]` looks sad ;P